app/controllers/app/freestyle_controller.rb

```diff
@@ -0,0 +1,26 @@
+class App::FreestyleController < App::BaseController
+  
+  # This line queries the database for all the different values of the body part attribute and stores them
+  FILTER_OPTIONS = Exercise.distinct.pluck(:body_part).freeze
+
+  # renders all the exercises with that particular body part
+  def index
+   apply_filter(params[:filter_by]) if params[:filter_by].present?
```

Comment:
It seems this `if` is redundant, since the `case` statement will not match with `nil`.